javascript-functions-practice/get-area-of-circle.js

```diff
@@ -0,0 +1,5 @@
+/* exported getAreaOfCircle */
+function getAreaOfCircle(radius) {
+  return (radius * radius) * 3.141592653589793;
```

Comment:
There's also a method you can use here, so you don't need to manually type it out, `Math.PI`. What you did isn't wrong, I just wanted to show you another way!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI

Reply:
@shawnkost That's good to know, thanks!